docs/src/main/asciidoc/kafka.adoc

```diff
@@ -1157,7 +1169,7 @@ mp.messaging.outgoing.fruit-out.value.serializer=io.quarkus.kafka.client.seriali
 
 Now, your Kafka messages will contain a JSON-B serialized representation of your Fruit pojo.
 
-NOTE: If you don't want to create a deserializer for each of your pojo, you can use the generic `io.vertx.kafka.client.serialization.JsonObjectDeserializer`
+NOTE: If you don't want to create a deserializer for each POJO, you can use the generic `io.vertx.kafka.client.serialization.JsonObjectDeserializer`
```

Comment:
Maybe we can replace all mentions of "pojo" with something else ? bean ? @cescoffier ?

Reply:
I would say data object. Not bean (Pojo are not necessarily managed, beans are)

Reply:
Could just use "object", we hardly have any other objects than POJOs these days :-)

Reply:
Data object is nice, done that.